tests/crypto/mbedtls_psa/testcase.yaml

```diff
@@ -34,8 +32,11 @@ tests:
     # Pick a platform which does not have an entropy driver. In this case we
     # enable the timer random generator because it's always available on all
     # platforms.
+    # Explicitly select CONFIG_MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG because this is
+    # not "automatically selected" when there is CSPRNG available.
```

Comment:
Shouldn't it be: when there is **no** CSPRNG available in the last part of this comment?

Reply:
You're absolutely right. Thanks